modules/transcription-service-microsoft-azure/src/main/java/org/opencastproject/transcription/microsoftazure/MicrosoftAzureTranscriptionService.java

```diff
@@ -280,6 +286,24 @@ public void activate(ComponentContext cc) {
       logger.info("Default '{}' audio encoding will be used", compressedAudioFormat);
     }
 
+    // If text should be split into multiple cues
+    Option<String> isSplitTextOpt = OsgiUtil.getOptCfg(cc.getProperties(), SPLIT_TEXT);
+    if (isSplitTextOpt.isSome()) {
+      splitText = Boolean.valueOf(isSplitTextOpt.get());
+    }
+    logger.info("Long text will be split: {}", splitText);
+
+    // When splitting text into multiple cues, how many characters should each cue have at most
+    Option<String> splitTextLineSizeOpt = OsgiUtil.getOptCfg(cc.getProperties(), SPLIT_TEXT_LINE_SIZE);
+    if (splitTextLineSizeOpt.isSome()) {
+      try {
+        splitTextLineSize = Integer.parseInt(splitTextLineSizeOpt.get());
+      } catch (NumberFormatException e) {
+        // Use default
+        logger.warn("Invalid configuration for split text line size. Default used instead: {}", splitTextLineSize);
```

Comment:
Please don't do that. Please let the service fail instead. If an admin misconfigures a service, the service should not just do something else. The admin clearly trued to set a specific setting and would probably assume everything is well if things seem to work. That is, until he later notices that the code ignored his setting.

Reply:
Did not think about, thanks for pointing it out!